src/javascript/app/pages/user/metatrader/metatrader.config.js

```diff
@@ -19,29 +19,29 @@ const MetaTraderConfig = (() => {
             const standard_config = {
                 account_type: 'standard',
                 leverage    : 1000,
-                short_title : localize('Standard'),
+                short_title : localize('Financial'),
             };
             const advanced_config = {
                 account_type: 'advanced',
                 leverage    : 100,
-                short_title : localize('Advanced'),
+                short_title : localize('Financial Pro'),
             };
             const volatility_config = {
                 account_type: '',
                 leverage    : 500,
-                short_title : localize('Synthetic Indices'),
+                short_title : localize('Synthetic'),
             };
 
             return ({
                 gaming: {
-                    demo_volatility: { mt5_account_type: volatility_config.account_type, max_leverage: volatility_config.leverage, title: localize('Demo Synthetic Indices'), short_title: volatility_config.short_title },
-                    real_volatility: { mt5_account_type: volatility_config.account_type, max_leverage: volatility_config.leverage, title: localize('Real Synthetic Indices'), short_title: volatility_config.short_title },
+                    demo_volatility: { mt5_account_type: volatility_config.account_type, max_leverage: volatility_config.leverage, title: localize('Demo Synthetic'), short_title: volatility_config.short_title },
+                    real_volatility: { mt5_account_type: volatility_config.account_type, max_leverage: volatility_config.leverage, title: localize('Real Synthetic'), short_title: volatility_config.short_title },
```

Comment:
@negarn sorry, `volatility_config`|`standard_config`|`advanced_config`. either in this or next pr, we need to rename all this to not make more confusion 🙏

Reply:
@mustofa-binary I think we should keep the variable naming, because it's what everyone knows, and if we name it financial it will cause confusion with MF accounts. so just updating the display names should be enough IMO.